docs/Rapid/3-User Manual/2-Explorer/2-Items/4-item-history/4-item-history.md

```diff
@@ -0,0 +1,59 @@
+
+### Viewing a previous version of an item
+
+A new drop-down menu appears that displays the current historical version of an item that is being viewed. Additionally, a new warning label appears to explain that you are viewing a historical item.
+
+![A screenshot that shows how an item's details page is displayed when viewing an item's history. A new dropdown menu explains what version is being viewed, and an information box explains how to undo any historical version changes, if desired. The screenshot is annotated with two red boxes that indicate these two sections of the details page.](<Item History Labels.png>)
+
+### Reverting an item back to its current version
+
+Pressing the **Undo Changes** button while viewing a historical view will revert the contents of the item back to its current state. Please note that an item's history does not track changes to the *Universal links* linked to an item.
+
+## Item History Components
+
+### Item Timeline
+
+By default, items in Rapid will have an **Item Timeline** component on the righthand side of the **Item Page**.
+
+![A screenshot that demonstrates how the Item Timeline appears. In this example, the timeline has two entries. The top entry states that the Status of the task was updated from "Note Started" to "In Progress". The lower entry states that the task was created. Both entries state the date and time for their respective entries.](<Items Timeline.png>)
+
+The **Item Timeline** component lists changes to an item, with the most recent changes appearing at the top of the component. Each modification to an item is accompanied by the name of the user who made the change, what the change was, and the exact time and date that the change was made.
+
+If this component has been removed by your system administrator, you can also view a timeline of changes to an item by navigating to the **Activity** tab in the tabstrip at the top of the page. This tab will provide details regarding the item's creation, edits, and any notes added by users as well. Note that the **Activity Feed** component contains more data for users. You can learn more about this component at the link below.
+
+![A screenshot demonstrating the functionality of the Ativity Feed. Its functionality is described in the section above. All changes are outlined for the user.](<Items Activity Tab.png>)
+
+- [The Activity Feed](</docs/Rapid/3-User Manual/2-Explorer/3-Pages/2-Page Components/Activity Feed Component/Activity Feed Component.md>)
+
+### Item Footers
+
+At the bottom of the **Item Page** is a footer, which contains important information about item history and versioning.
+
+![A screenshot that shows the location of the footer on an item page. The screenshot is annotated with a red box and arrow to highlight the footer's location. The footer in this example reads: "Employee 3" then a large gap followed by "Created April 11, 2024 by <censored> | Last Updated April 11, 2024 by <censored>.](<Items Footer Location.png>)
```

Comment:
Turns out the <censored> gets interpreted as a HTML element by the parser causing it to not build